### ISSUE TITLE
PDF viewer produces internal server error

### ISSUE THREAD

**Issue by zarquon42b** [ckan/ckanext-pdfview#5]
Hi,
we upgraded to CKAN 2.3 yesterday and installed this extension. Unfortunately, the preview gives us an internal server error: 
![publication - unnamed resource - rdmba - chromium_044](https://cloud.githubusercontent.com/assets/1515863/6710682/8ab9b0f2-cd82-11e4-9d79-46554f8d8b91.png)

the corresponding output in the log is:

```
Wed Mar 18 15:22:21 2015] [error] [client 132.xxx.xxx.xxx] Error - <type 'exceptions.AttributeError'>: 'module' object has no attribute 'ckanext-pdfview/main'
```

Is this a configuration issue on our side?

Best
Stefan

**Comment by jqnatividad** [comment]
@zarquon42b, did you install using pip?  If so, the PyPI release is yet to be updated with the latest code.

Do a Development Installation for now and it should work.

**Comment by zarquon42b** [comment]
Same result.

**Comment by zarquon42b** [comment]
sorry, my bad. I t works now.
Thanks a lot.

**Comment by rossjones** [comment]
@zarquon42b did you do anything? I'm getting 

`AttributeError: 'module' object has no attribute 'ckanext-pdfview/main'`

when I try and use the pdf_view

---

Edit: python setup.py develop instead of python setup.py install (on a cloned version) worked for me.

**Comment by zarquon42b** [comment]
At first, installing the version from git repo did not work at all. But suddenly,  after some fiddeling about, removing all remains of pdfview from the site-library and rerunning `python setup.py develop` in pdfview's directory, it suddenly worked. I also cleaned ckan directory using git clean -f -x -d and reran `python setup.py develop`.

I cannot explain why it suddenly worked.

Best Stefan

**Comment by jqnatividad** [comment]
Thanks @zarquon42b, @rossjones.  I suspect this is because [pdfview used to be part of CKAN core](https://github.com/ckan/ckan/pull/2270), and both of you apparently did upgrades of your existing pre-2.3 source installations.  

On a fresh 2.3 install it works.

I'll add a Troubleshooting section to the README for other upgraders.

**Comment by amercader** [comment]
This was most likely caused because we were not including the fanstatic `resource.config` file when packaging the extension. This was fixed in 63e3bf275363

Can people do `pip install --upgrade ckanext-pdfview` to see if the issue is fixed?

Sorry about the confusion, I'll update the extensions template on ckan core to make sure this file is included.

@jqnatividad I uploaded the new version to Pypi, let me know your user name there and I'll add you as a maintainer so you can upload new versions.

I've re-added the instructions for releasing a new version: https://github.com/ckan/ckanext-pdfview#releasing-a-new-version-of-ckanext-pdfview

**Comment by jqnatividad** [comment]
Thanks @amercader!  My Pypi username is the same (jqnatividad)
